io.github.zzzgydi.ClashVerge.yml

```diff
@@ -0,0 +1,83 @@
+app-id: io.github.zzzgydi.ClashVerge
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: clash-verge
+rename-icon: clash-verge
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --persist=.config/clash-verge
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
```

Comment:
Is it not possible to get rid of these two permissions?

Reply:
No, software will not appear on the tray without these two permissions

Reply:
These *two*, or this *one*? I don't see how `org.freedesktop.Notifications` is relevant for the systray, when `org.freedesktop.portal.Notification` exists, and this is completely unrelated to the systray.

Only the KDE permission should *maybe* be necessary.